js/xt.js

```diff
@@ -0,0 +1,1072 @@
+        //         "symbol": "btc_usdt",
+        //         "contractType": "PERPETUAL",
+        //         "underlyingType": "U_BASED",
+        //         "contractSize": "0.0001",
+        //         "tradeSwitch": true,
+        //         "state": 0,
+        //         "initLeverage": 10,
+        //         "initPositionType": "CROSSED",
+        //         "baseCoin": "btc",
+        //         "quoteCoin": "usdt",
+        //         "baseCoinPrecision": 8,
+        //         "baseCoinDisplayPrecision": 8,
+        //         "quoteCoinPrecision": 8,
+        //         "quoteCoinD        let method = 'publicGetDataApiV1GetMarketConfig';rustType": "TAKE_PROFIT,STOP,TAKE_PROFIT_MARKET,STOP_MARKET,TRAILING_STOP_MARKET",
+        //         "supportPositionType": "CROSSED,ISOLATED",
+        //         "minPrice": null,
+        //         "minQty": "1",
+        //         "minNotional": "5",
+        //         "maxNotional": "100000",
+        //         "multiplierDown": "0.005",
+        //         "multiplierUp": "0.005",
+        //         "maxOpenOrders": 200,
+        //         "maxEntrusts": 120,
+        //         "makerFee": "0.00015",
+        //         "takerFee": "0.0007",
+        //         "liquidationFee": "0.015",
+        //         "marketTakeBound": "0.005",
+        //         "depthPrecisionMerge": 5,
+        //         "labels": [
+        //         "HOT"
+        //         ],
+        //         "onboardDate": 1654857001000,
+        //         "enName": "BTCUSDT Perpetual",
+        //         "cnName": "BTCUSDT 永续",
+        //         "minStepPrice": "0.5"
+        //       },
+        //    ]
+        //
+        // COIN
+        //
+        //  [
+        //    {
+        //      "symbol": "btc_usd",
+        //      "contractType": "PERPETUAL",
+        //      "underlyingType": "COIN_BASED",
+        //      "contractSize": "100",
+        //      "tradeSwitch": true,
+        //      "state": 0,
+        //      "initLeverage": 10,
+        //      "initPositionType": "CROSSED",
+        //      "baseCoin": "btc",
+        //      "quoteCoin": "usd",
+        //      "baseCoinPrecision": 8,
+        //      "baseCoinDisplayPrecision": 8,
+        //      "quoteCoinPrecision": 8,
+        //      "quoteCoinDisplayPrecision": 8,
+        //      "quantityPrecision": 0,response
+        //      "maxEntrusts": 120,
+        //      "makerFee": "0.00015",
+        //      "takerFee": "0.0007",
+        //      "liquidationFee": "0.015",
+        //      "marketTakeBound": "0.005",
+        //      "depthPrecisionMerge": 5,
+        //      "labels": [
+        //      "HOT"
+        //      ],
+        //      "onboardDate": 1654859400000,
+        //      "enName": "BTCUSD Perpetual",
+        //      "cnName": "BTCUSD 永续",
+        //      "minStepPrice": "0.5"
+        //    },
+        //  ]
+        //
+        const marketsUSDT = await this.parseSwapAndFutureMarkets (responseUSDT);
```

Comment:
Why not concatenate both raw responses and call `parseSwapAndFuturesMarkets` only once?